Bugfix/update sphinx documentation

louisegrimble opened the pull request:
# Pull Request Template

## Summary
**What does this pull request address? Provide a brief description.**
This PR makes a number of updates to the sphinx documentation. Firstly ensuring the solosis command is written as `solosis-cli`, adding some inital commands to the homepage to allow users to make a "quickstart". Finally to add examples of commands with an actual sample and give expected outcomes so that users can see whether their samples are working as intended. 

## Changes Made
**List major changes or highlight key points:**
- Fixed the main solosis command from `cli` to `solosis-cli`. #71 
- added "quick start" 3 main commands to index.rst (home) page #76 
- added real examples for each command on examples page (examples.rst) #77 

*NOTE - ALL CHANGES WHERE MADE BEFORE COMMITING SO ALL CHANGES CAN BE SEEN IN THE PR ARE GOING TO BE LINKED TO #71 BUT I HAVE MENTIONED ABOVE WHICH FILES HAVE CHANGES TO WHICH ISSUE- APOLOGIES FOR THIS!!*

## Checklist
- [x] The code adheres to the project's style guidelines
- [x] Documentation has been updated (if applicable)
- [ ] Tests have been added/updated (if applicable)
- [x] I have linked this PR to an issue (if applicable)
- [x] I have reviewed my own code for potential issues



## Additional Information (Optional)
**Include any relevant context, links, or references.**  
Pages to view the changes made:
https://haniffalab.com/solosis/index.html 
https://haniffalab.com/solosis/examples.html

closes #71 
closes #76 
closes #77

davehorsfall reviewed (changes requested):
The `input.rst` file is a little confusing. I think the top section can be removed entirely. 

I like the idea of having a standard metadata input file, and this can be documented in `input.rst`. However, some commands will inevitably deviate from this format for various reasons. This page should just clarify the expected formats and required columns for the commands available in solosis.

louisegrimble commented:
There are still changes to be made to this branch before merging the PR. Expected to be commited by 24/01/2025

louisegrimble commented:
Changes made based on comments and other discussions this week:
- Removed the run.rst file as it was no longer needed 
- Updated the input page to be accurate to recent updates to solosis 
- Updated output page, to add expected output paths and directory names, also including tree diagrams.
- Updated setup page to include environment variables and loading and login into irods (if using irods commands).
